pages/common/jhipster.md

```diff
@@ -0,0 +1,36 @@
+# jhipster
+
+> Generates a monolithic or microservices project after prompting user.
+> More information: <https://www.jhipster.tech/>.
+
+- Generate a simple full-stack project (monolithic or microservices):
+
+`jhipster`
+
+- Generate a simple frontend project (with OPTION = --skip-server) or a simple backend project (with OPTION = --skip-client):
+
+`jhipster [OPTION]`
+
+- Upgrade a generated project with the last JHipster updates:
+
+`jhipster upgrade`
+
+- Add a new entity to a generated project:
+
+`jhipster entity {{entity_name}}`
+
+- Import a JDL file to configure your application (entities...) - See: https://start.jhipster.tech/jdl-studio/:
+
+`jhipster import-jdl {{first_file.jh}} {{second_file.jh}}... {{n_file.jh}}`
```

Comment:
I have doubts with the usage of the `...` in this example, but I don't remember how we should address this, so it is possible another mate might help with this.

Reply:
Yep I've searched about it in man pages and it seems it's formatted that way:

```
ls [OPTION]... [FILE]...
```

Reply:
```suggestion
`jhipster import-jdl {{first_file.jh second_file.jh ... n_file.jh}}`
```
Maybe like this, but not sure.

Reply:
Yep it seems it's like it by searching on the other pages with `\{.*\.\.\.\}`